graph/src/log/elastic.rs

```diff
@@ -6,9 +6,11 @@ use std::sync::{Arc, Mutex};
 use std::time::Duration;
 
 use chrono::prelude::{SecondsFormat, Utc};
+use futures::future;
 use futures::{Future, Stream};
 use itertools;
 use reqwest;
+use reqwest::r#async::Client;
```

Comment:
Where can I read more about this language feature? What is it called?

Reply:
https://doc.rust-lang.org/edition-guide/rust-2018/module-system/raw-identifiers.html